server/methods/catalog.js

```diff
@@ -832,7 +832,8 @@ Meteor.methods({
     let update;
     // handle booleans with correct typing
     if (value === "false" || value === "true") {
-      update = EJSON.parse(`{${field}:${value}}`);
+      const booleanValue = (value === "true" || value === true);
```

Comment:
the second half of this evaluation seems unnecessary. 
Because we're filtering for either `"false"` or `"true"` in the `if` statement above, `value` can never equal `true`.